helpers-es5.js

```diff
@@ -187,7 +188,6 @@ var path = require('path'),
                     print('Files:create', 'Creating file: ' + name);
                     if (name === 'manifest.json') {
                         properties.name = options.appName;
-                        properties.short_name = options.appName;
```

Comment:
Ahh I see. I think someone PR'ed `properties.short_name = options.appName;` into `helpers-es5` but not `helpers.js`. Do you reckon you could add it to the ES6 version then re-compile? Thanks mate :)